incubator/nfs-client-provisioner/templates/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
I would say, a storage class should not be part of the chart. It is usually not chart-specific and should be managed externally. Also, as far a I can see, the storage class is not used. I don't see any PVC.

See also #1869. This pattern should be applied.

Reply:
@unguiculus I included the storage class into the chart, because each instance of such a nfs storage class is tied to an instance of the provisioner. Therefore you have a strong coupling between these two. This is not ideal, because the storage class cannot be changed afterwards.

I'm open to suggestions how to handle this. Maybe an output from NOTES.txt after installation?

Reply:
OK, I think I understand now how this things works.